package.json

```diff
@@ -141,6 +141,7 @@
     "@typescript-eslint/eslint-plugin": "^5.59.11",
     "@typescript-eslint/parser": "^5.59.11",
     "audit-ci": "^6.6.1",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
Added `cheerio` as a dev dependency. This library is what we use in the unit tests that test the logic in the nunjucks templates (see below 👇 )